pkg/api/types.go

```diff
@@ -204,6 +204,8 @@ type VolumeSource struct {
 
 // Similar to VolumeSource but meant for the administrator who creates PVs.
 // Exactly one of its members must be set.
+// All PersistentVolumes are volumes, only some Volumes are persistent.  All pointers
```

Comment:
In that case, do we need an internal type to resolve the standoff between VS and PVS?  Refactor the plugins to work against the internal type.  The API types translate to it.  Is this a weird thing to express in the converters and internal types.go file?